nc/include/component/MeshCollider.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "Component.h"
+
+namespace nc
+{
+    class MeshCollider final : public ComponentBase
```

Comment:
I've changed my mind. I prefer ConcaveCollider over MeshCollider. Years down the road, when we've forgotten about the fact that this should only be a concave collider and have performance issues, we're going to wish we'd named it such.